examples/message/tests/provider/test_message_provider.py

```diff
@@ -49,3 +65,19 @@ def test_verify_failure_when_a_provider_missing():
     with pytest.raises(AssertionError):
         with provider:
             provider.verify()
+
+
+def test_verify_from_broker(default_opts):
+    provider = MessageProvider(
+        message_providers={
+            'A document created successfully': document_created_handler,
+        },
+        provider='ContentProvider',
+        consumer='DetectContentLambda',
+        pact_dir='pacts'
+
+    )
+
+    with pytest.raises(AssertionError):
```

Comment:
What is this testing?

Reply:
Hi @elliottmurray,
Thank you to catch this issue, I'm currently changing the test to run with the pact-broker as we do for the e2e testing.
I'll update the pull request asap. F.